### ISSUE TITLE
Add GitHub workflow to automatically update to newer JupyterLab versions

### ISSUE THREAD

**Issue by jtpio** [jupyter/notebook#7271]
To ease maintenance, it would be useful to have a script that checks what the latest version of JupyterLab is, and automatically opens a PR that bumps to the newer packages.

Some ideas and possible config options for that script:

- run as a cron, and / or via a workflow dispatch on demand
- support updating to a specific version, `latest` (stable) or `next` (pre-release)
- this might also require being able to specify a branch, as multiple versions of notebook (stable and pre-release) can be developed at the same time

JupyterLab Desktop has something similar: https://github.com/jupyterlab/jupyterlab-desktop/pull/311

**Comment by jtpio** [comment]
In some cases there is more work to do than just updating the versions (for example integrating some upstream plugins more natively in Notebook). But at least this script would help with the repetitive parts.

**Comment by itsmevichu** [comment]
@RRosio @jtpio Can we close this issue?

**Comment by jtpio** [comment]
Right, there is now a workflow, but it currently needs to be triggered manually (I've been using it from the fork for the past few months).

**Comment by krassowski** [comment]
The approach from https://github.com/jupyterlab/jupyterlab-desktop/blob/master/.github/workflows/sync_lab_release.yml should work here too, it requires setting up a new app.
